### ISSUE TITLE
Stop assuming that expressions are on a single line

### ISSUE THREAD

**Issue by 0xekez** [ZekeMedley/lust#1]
Reasonably low priority but needs to be done before this can properly parse files and as a precursor for a decent REPL

**Comment by 0xekez** [comment]
This is coming along nicely now in the cleanup branch.

**Comment by 0xekez** [comment]
Done with merge of `cleanup` branch.
